daemon/daemon_windows.go

```diff
@@ -261,9 +261,12 @@ func (daemon *Daemon) initNetworkController(config *Config, activeSandboxes map[
 		}
 
 		if !found {
-			err = v.Delete()
-			if err != nil {
-				return nil, err
+			// overlay networks are global and should not be deleted by local HNS
+			if v.Type() != "overlay" {
```

Comment:
Instead of v.Type() variable, you can use v.Info().Scope() constant and compare it against `datastore.GlobalScope` or simple "global" string.